android/app-example/src/main/java/com/badoo/ribs/example/rib/dialog_example/builder/DialogExampleScope.kt

```diff
@@ -0,0 +1,7 @@
+package com.badoo.ribs.example.rib.dialog_example.builder
+
+import javax.inject.Scope
+
+@Scope
+@Retention(AnnotationRetention.RUNTIME)
```

Comment:
retention is runtime by defaut

Reply:
This is coming from the generated templates, we can change it there. Otherwise is it a problem to be explicit with it?

Reply:
I've left a comment about it in the first app-example iteration https://github.com/badoo/RIBs/pull/1/files#r258449002
I think it is better to remove it because now it may look like you are changing the default retention for some reason but in fact it doesn't change anything.

Reply:
Let's move this to the discussion about templates then.